### DISCUSSION TITLE
JackTrip as local "Audio over ethernet" solution

### DISCUSSION THREAD

**Discussion by johannphilippe** [jacktrip/jacktrip#840] [Howtos]
Dear JackTripers, 
I'm an electroacoustic composer working with Jacktrip. I had successful results using it as a solution to share 16 in/out channels between two musicians over an ethernet local network : with a bit of configuration, we could perform without audible latency and without any packet being lost (at least, no audio clics). 
Though, more recently, I could not achieve the same. With exact same config, a lot of packets are lost. I tried a lot of parameters change, no result. 
What settings would you recommand on a local network (with GB Ethernet switch) ? Are there some points I should primarily check to avoid such packet loss ? 
The project here is to use jacktrip to route ~=30 channels from/to different performers locally.

**Comment by cchafe** [comment]
Most core features haven't changed in a very long time. What's your OS, and can you detail an example server and client command invocation? Maybe we can test the same settings.

**Comment by johannphilippe** [comment]
I still use the same version so i don't suspect jacktrip to be responsible, just looking for some piece of advice ! 

Server is on Linux mint 20 with following command : 
`jacktrip -S -p4 -r6 -q8` (after tests, it was the best I could experiment when it worked for the previous conncert).
Clients are two Mac ( Monterey) + a debian machine 
`jacktrip -C 192.168.1.??`

We tried a lot of setups for "-q" and "-a", difficult to say which one is best...
Also I could not determine if low vector size is better or not, since I'm not an expert in networks, so we tried from 64 to 512 with similar results (sometimes a bit more clics, sometimes similar).

**Comment by cchafe** [comment]
The only  thing is the "-r 6" which is quite extreme. That will create a stream that is 6x the bandwidth. Redundancy is rarely used and only to mitigate a network path where there's difficult packet loss and lots of spare bandwidth. That's an unusual situation.

**Comment by johannphilippe** [comment]
We will then try without redundancy. And I'll let you know. 
Thank you so much for answering !
